mobilecoind/src/processed_block_store.rs

```diff
@@ -200,6 +224,18 @@ impl ProcessedBlockStore {
             )?;
         }
 
+        for utxo in spent_utxos.iter() {
+            let processed_tx_out = ProcessedTxOut::from_spent_utxo(utxo);
+            let processed_tx_out_bytes = mc_util_serial::encode(&processed_tx_out);
+            println!("store {:?}", utxo.key_image);
```

Comment:
Should this use a logger or does this automatically redirect to the correct logger?

Reply:
It shouldn't be there at all :)